setup.cfg

```diff
@@ -5,3 +5,16 @@ versionfile_source = napari/_version.py
 versionfile_build = napari/_version.py
 tag_prefix = v
 parentdir_prefix = napari-
+
+[flake8]
+# Ignores - https://lintlyci.github.io/Flake8Rules
+# E203  Whitespace before ':'  (sometimes conflicts with black)
+# E501 line too long (84 > 79 characters)  (sometimes too annoying)
+# W503	Line break occurred before a binary operator
+# E711	Comparison to none should be 'if cond is none:'
+# E712	Comparison to true should be 'if cond is true:' or 'if cond:'
+# C901 McCabe complexity test. Would be nice to reenable, but takes work
```

Comment:
Never heard of this one! I quite like it! Yes, I envision making multiple (many?) smaller PRs to get this fixed. I wouldn't look forward to reviewing a huge one that tackles it for the whole library!

Reply:
if by "this one" you mean the mccabe test, I think we're actually in amazingly good shape that there are only two overly complex methods (on `Shapes`).  probably won't take long for someone to split those into a few methods and pass that test.